test/DynamoCoreWpfTests/ViewExtensions/PythonMigrationViewExtensionTests.cs

```diff
@@ -353,6 +353,5 @@ public void IronPythonPackageLoadedTest()
             Assert.AreEqual(GraphPythonDependencies.PythonPackage, pkgDependencyInfo.Name);
             Assert.AreEqual(GraphPythonDependencies.PythonPackageVersion, pkgDependencyInfo.Version);
         }
-
```

Comment:
@reddyashish could you simply revert this file if it isn't involved in this PR.

Reply:
Yes, just thought I would clean this up as well because of the unnecessary extra new line.